src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig

```diff
@@ -57,82 +64,167 @@
         </tr>
       </thead>
       <tbody>
-        {% for product in orderForViewing.products.products %}
-          <tr>
-            <td>
-              {% if product.imagePath %}
-                <img src="{{ product.imagePath }}" alt="{{ product.name }}">
-              {% endif %}
-            </td>
-            <td>
-              <a href="{{ path('admin_product_form', {'id': product.id}) }}">
-                <p class="mb-0">{{ product.name }}</p>
-                {% if product.reference %}
-                  <p class="mb-0">
-                    {{ 'Reference number:'|trans({}, 'Admin.Orderscustomers.Feature') }}
-                    {{ product.reference }}
-                  </p>
-                {% endif %}
-                {% if product.supplierReference is not empty %}
-                  <p class="mb-0">
-                    {{ 'Supplier reference:'|trans({}, 'Admin.Orderscustomers.Feature') }}
-                    {{ product.supplierReference }}
-                  </p>
-                {% endif %}
+        {% for product in orderForViewing.products.products|slice(0, paginationNum) %}
```

Comment:
Same here, please add `@todo` statement to remove this array_slice 😉

Reply:
:ballot_box_with_check: Thanks